src/main/java/game/Effect.java

```diff
@@ -18,23 +18,46 @@
     private List<String> inventoryAddChange;
     private List<String> inventoryRemoveChange;
 
+    public Effect(String description) {
+        this.description = description;
+    }
+
+    public Effect(String description, int sceneIdChange) {
+        this(description);
+        this.type = Type.NAVIGATION;
+        this.sceneIdChange = sceneIdChange;
+    }
+
+    public Effect(String description, Stats statsChange) {
+        this(description);
+        this.type = Type.STATS;
+        this.statsChange = statsChange;
+    }
+
+    public Effect(String description, List<String> inventoryAddChange, List<String> inventoryRemoveChange) {
+        this(description);
+        this.type = Type.INVENTORY;
+        this.inventoryAddChange = inventoryAddChange;
+        this.inventoryRemoveChange = inventoryRemoveChange;
+    }
+
     /**
      * Applies the changes specified effect to the game.
      * @param game the Game object to apply this effect on
      */
-    public void apply(Game game) {
+    public void apply(Game game, Interactable.Callback callback) {
         if (game == null) throw new IllegalArgumentException("game is null");
 
         switch (type) {
```

Comment:
In switch statements it is usually a good practice to add a default in order to set up another break point in case another part of your system fails to pass the right data to this method.